coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -17,6 +17,9 @@ def to_camelcase(string):
     :param string: The string to convert.
     :return:       The camel-cased string.
     """
+    string = re.sub("(\s)",
+                    lambda match: '_',
+                    string)
```

Comment:
why do you introduce those lines? They don't add the new algorithm so they don't belong into this commit right? (You can use `git reset master` and `git add -p` to make new commits out of the parts.)